libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -178,6 +178,8 @@ Json::Value ASTJsonConverter::inlineAssemblyIdentifierToJson(pair<yul::Identifie
 	tuple["declaration"] = idOrNull(_info.second.declaration);
 	tuple["isSlot"] = Json::Value(_info.second.suffix == "slot");
 	tuple["isOffset"] = Json::Value(_info.second.suffix == "offset");
+	tuple["isSelector"] = Json::Value(_info.second.suffix == "selector");
+	tuple["isAddress"] = Json::Value(_info.second.suffix == "address");
```

Comment:
This adds these values for all identifiers. Shouldn't we limit it only to variables of function types? Or do we intentionally want all these properties present for all identifiers? It seems like it would add a lot of noise though.

Reply:
Maybe we should find a more general solution here anyway? On the other hand, we now use `.selector` and `.address` instead of `_selector` / `_address`. I think it is still not unique since you might use an identifier path to access scopes.

Maybe it would suffice to say "the suffix is used to access a 'property' instead of a scoped variable"? Which property it is can be derived from the suffix itself.

Reply:
Can we fix this before we forget it?